app/models/media.rb

```diff
@@ -1,5 +1,10 @@
 # The Media resource manages file uploads to the platform
 class Media < ApplicationRecord
+  # 16:9 of 1290 is 1290:725.625
+  # We rounded up.
+  # @see https://www.ios-resolution.com/
+  FULL_WIDTH_16_BY_9 = [1290, 726]
```

Comment:
We have full-width on smol devices; which is fine for the `Product::Card` and `SectionNavigation::SectionCard`, but probably not ideal when rendering the `Section` itself on a larger monitor.

I think it's fine-for-now to just be :this-is-fine:, but I do hope we'll figure out a way to follow up on this so hero-images-on-laptop-sized devices with retina displays are also big and beautiful.

Perhaps this is a place where media-queries will need to come into play; especially since we don't want to load  really-big-photos on a really-smol-devices.

All that said, Mobile First is probably the way to play this; and we can polish up the presentation on medium/large/xlarge screens :later: